Dockerfile

```diff
@@ -18,8 +18,11 @@ ARG gid=1000
 RUN groupadd -g ${gid} ${group} \
     && useradd -d "$JENKINS_HOME" -u ${uid} -g ${gid} -m -s /bin/bash ${user}
 
-RUN mkdir -p $JENKINS_HOME/init.groovy.d
+RUN mkdir -p $JENKINS_HOME/init.groovy.d && \
```

Comment:
nit: for readability - the convention that I've seen on official Dockerfiles up on dockerhub is to put the "\\\" on the first line, and the "&&" on the continuing line. 
Couple references:
https://hub.docker.com/r/selenium/node-firefox/~/dockerfile/
https://github.com/docker-library/python/blob/1ca4a57b20a2f66328e5ef72df866f701c0cd306/2.7/Dockerfile